src/main/java/nl/knaw/dans/pf/language/emd/validation/EMDValidator.java

```diff
@@ -38,7 +38,7 @@
      */
     public static final String VERSION_0_1 = "0.1";
 
-    public static final String SCHEMA_LOCATION = "http://easy.dans.knaw.nl/schemas/md/emd/2018/04/emd.xsd";
+    public static final String SCHEMA_LOCATION = "http://easy.dans.knaw.nl/schemas/md/emd/2019/01/emd.xsd";
```

Comment:
shouldn't this be newer as well? You expanded the EMD with a new field, after all...